binding.gyp

```diff
@@ -71,6 +71,17 @@
           ],
         }],
       ]
+    },
+    {
+      "target_name": "action_after_build",
```

Comment:
rather than adding a 2nd compile target and linking back the original compiled binary, I think it would be better to add an optional condition passed at the command line during an npm install.